added location path of mplog file (#8807)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/troubleshoot-performance-issues.md

We recommend that software vendors follow the various guidelines in [Partnering with the industry to minimize false positives](https://www.microsoft.com/security/blog/2018/08/16/partnering-with-the-industry-to-minimize-false-positives/). The vendor can submit their software through the [Microsoft Security Intelligence portal](https://www.microsoft.com/wdsi/filesubmission?persona=SoftwareDeveloper).

## Analyze the Microsoft Protection Log

> Location of Microsoft Protection Log file can be found in **C:\ProgramData\Microsoft\Windows Defender\Support**

Reviewer: ```suggestion
You can find the Microsoft protection log file in **C:\ProgramData\Microsoft\Windows Defender\Support**.
```
